jaxrs-tck/src/test/java/jakarta/ws/rs/tck/common/matchers/IsThrowingMatcherTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2022 Jeremias Weber. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+package jakarta.ws.rs.tck.common.matchers;
+
+import org.junit.jupiter.api.Test;
+
+import static ee.jakarta.tck.ws.rs.common.matchers.IsThrowingMatcher.isThrowing;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.not;
+
+final class IsThrowingMatcherTest {
```

Comment:
If you add @author then everbody will give credits to you. ;-)

Reply:
Yeah but in other projects people discouraged me from adding myself as an author if I had nothing else to say in the JavaDoc. That's why I didn't add that here.

Reply:
You can safely ignore them. It is your right to do that, so if you want, just do it.

Reply:
If you convince me to change the test of the matcher and I'm gonna have to change my code anyway, I think I'll add it. If not (and believe me I'm stubborn on this 😉), then I'm just gonna leave it as is.